src/tracks/playabletrack/wavetrack/ui/WaveTrackView.cpp

```diff
@@ -1098,6 +1098,33 @@ auto WaveTrackView::GetSubViews(const wxRect* rect) -> Refinement
    return results;
 }
 
+unsigned WaveTrackView::CaptureKey(wxKeyEvent& event, ViewInfo& viewInfo, wxWindow* pParent, AudacityProject* project)
+{
+    if (auto affordance = GetAffordanceControls())
```

Comment:
This function, and the two following, may cause. Return key, while the mouse is in a wave track, to edit a clip name.

Was that part of the design?

I think the more correct way to do call-through:  call through to the affordance, but then check event.GetSkipped().  If it is true, meaning the affordance did not take its first-chance to capture the key, then call through to the base class in case it wants to capture.

Take the bitwise OR of the return from the two captureKey calls, as the return value of the function.

It may not make a difference now, because nothing really happens in the base class methods, but don't assume that will remain so.

Make similar fixes in the next two functions.

Reply:
>Was that part of the design?

Yes, though editing will be invoked only if there is a "selected" clip on that track.